elements/lisk-tree/src/sparse_merkle_tree/constants.ts

```diff
@@ -19,8 +19,16 @@ export const EMPTY_HASH = hash(EMPTY_VALUE);
 
 export const LEAF_HASH_PREFIX = Buffer.from('00', 'hex');
 export const BRANCH_HASH_PREFIX = Buffer.from('01', 'hex');
+export const PREFIX_HASH_TO_VALUE = Buffer.from([0]);
+export const NODE_HASH_SIZE = 32;
```

Comment:
How did you derive this, Is it always 32 ?

Reply:
we are using SHA256 so it should always give out 32 byte hash